manticore/ethereum.py

```diff
@@ -2455,6 +2469,14 @@ def _generate_testcase_callback(self, state, name, message=''):
         def flagged(flag):
             return '(*)' if flag else ''
         testcase = self._output.testcase(name.replace(' ', '_'))
+
+        # call on_finalize for each plugin
+        for plugin in self.plugins:
+            try:
+                plugin.on_finalize(state, testcase)
+            except AttributeError:
```

Comment:
We shouldn't use a try except here as if the exception is thrown inside `on_finalize` we may not notice it and loose time debugging.

We should either assign method to variable and have try except over that (and call the method after the try except blocks) or use hasattr or getattr (with an empty function as a default value).